src/NexusMods.App.UI/Pages/LibraryPage/LibraryView.axaml.cs

```diff
@@ -55,13 +55,13 @@ public LibraryView()
             deactivate.Merge(activate)
                 .Where(static tuple => tuple.Model is LibraryItemModel)
                 .Select(static tuple => ((tuple.Model as LibraryItemModel)!, tuple.Item2))
-                .Subscribe(this, static (tuple, view) => view.ViewModel!.ActivationSubject.OnNext(tuple))
+                .Subscribe(this, static (tuple, view) => view.ViewModel!.Adapter.ModelActivationSubject.OnNext(tuple))
                 .AddTo(disposables);
```

Comment:
Could this and the lines above (from L45), be moved into a method on the Adapter? 
It is repeated code on every view using the new TreeDataGrid with the Adapter